### PR TITLE
Add Point>>#isInRectangle

### PR DESCRIPTION
**Pull request by Ducasse**
- Add tests
- Add isInRectangle:
- Add executable comments to containsPoint:
- Add executable comment to isInRectangle:
- Recategorise is...Triangle...

### CODE REVIEW

File: src/Kernel/Point.class.st [hunk 1]
```diff
@@ -397,7 +397,20 @@ Point >> isFloatPoint [
 	^ x isFloat and: [ y isFloat ]
 ]
 
-{ #category : #geometry }
+{ #category : #testing }
+Point >> isInRectangle: aRectangle [
```
Comment (astares) [inline src/Kernel/Point.class.st]: I would call it 

 #is**Inside**Rectangle:
 
 to be in alignment with
 
  #is**Inside**Circle:

File: src/Kernel/Point.class.st [hunk 2]
```diff
@@ -397,7 +397,20 @@ Point >> isFloatPoint [
...
+	"(0@0 isInRectangle: (0@0 corner: 100@100)) >>> true"
+	"(0@10 isInRectangle: (0@0 corner: 100@100)) >>> true"
+	"(100@100 isInRectangle: (0@0 corner: 100@100))>>> false"
+	"(100@90 isInRectangle: (0@0 corner: 100@100))>>> false"
+	
+	^ aRectangle containsPoint: self
+]
+
+{ #category : #testing }
 Point >> isInsideCircle: a with: b with: c [
```
Comment (astares) [inline src/Kernel/Point.class.st:401]: a, b, c as argument names is not very descriptive. Could we have a comment what they depict if one does not have the mentioned paper at hand?

Comment (Ducasse) [reply src/Kernel/Point.class.st:401]: I have no idea. This is not me that wrote it. 
I just moved it to the testing protocol.

File: src/Kernel/Point.class.st (hunk 1 above)
Comment (Ducasse) [reply src/Kernel/Point.class.st]: Good suggestion. I will fix this.

Comment (astares) [reply src/Kernel/Point.class.st]: Thanks!

**Comment by MarcusDenker** [comment]
Failing tests not related:

- [CI] Failing test: ProperMethodCategorizationTest>>testNoUncategorizedMethods #11669
- [CI] System variable environment on windows does not use adequate encodings #11407

**Pull request merged**